ui/src/components/AnalyzeButton/AnalyzeButton.tsx

```diff
@@ -1,6 +1,6 @@
 import {Button} from "@material-ui/core";
```

Comment:
can we remove this file? is it moved to common?

Reply:
i have a debate about it.
will it be used by both apps?